### ISSUE TITLE
decision(consistency): nukleros.io labels

### ISSUE THREAD

**Issue by scottd018** [nukleros/platform-component-library#4]
Need to make a decision for consistency regarding the nukleros.io/platform (and all other nukleros.io labels for that matter).  Do we want to label ALL things with these labels, or are we only concerned with application-like resources (e.g. deployments, pods, statefulsets, etc).

Additionally, although we are namespacing these off, we may want to use our own selector to prevent collisions in the instance that someone would `kubectl apply` the same project (e.g. cert-manager) to one of our namespaces.

TLDR:

- Decision on which resources to place the `nukleros.io` labels on (all vs. just application-like resources)
- Decision on whether to enforce (soft enforcement via code review) using the nukleros.io labels as part of the selectors to prevent collisions.

**Comment by scottd018** [comment]
Decision:

1. Place labels on **everything** that we deploy with a `platform.nukeros.io/group` and `platform.nukeros.io/project`:
  - group = logical description of what it is it a part of (external-dns = ingress, nginx = ingress)
  - project = where did this thing come from
2. Don't worry about danger of overlap - keep it simple
